iroh/src/magicsock/endpoint_map/endpoint_state.rs

```diff
@@ -934,6 +961,7 @@ impl EndpointStateActor {
             }
             self.open_path(addr);
             self.close_redundant_paths(addr);
+            self.pub_selected_path.set(Some(addr.clone().into())).ok();
```

Comment:
I don't follow why we store selected path twice?  Can't we use the watchable directly?  Or is that too horrible?